server/Controllers/RecipeDetailsController.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using server.Models;
+using Microsoft.EntityFrameworkCore;
+using server.Data;
+using server;
+
+namespace server.Controllers
+{
+  [ApiController]
+  [Route("api/[controller]")]
+  public class RecipeDetailsController : ControllerBase
+  {
+
+    private readonly ApplicationDbContext _context;
+    public RecipeDetailsController(ApplicationDbContext context)
+    {
+      _context = context;
+    }
+
+
+// Test function to check if recipe data is being returned or not
+    [HttpGet]
+    public async Task<IActionResult> Test()
```

Comment:
All of this should go under the `RecipeController`. "Details" are a subsection of recipes, much like searching, adding and deleting.